numpy/random/_pickle.py

```diff
@@ -33,8 +37,16 @@ def __bit_generator_ctor(bit_generator_name='MT19937'):
     else:
         raise ValueError(str(bit_generator_name) + ' is not a known '
                                                    'BitGenerator module.')
+    if seed_seq_info is None:
+        # Handles legacy pickles that do not preserve seed sequence info
+        return bit_generator()
 
-    return bit_generator()
+    # Unpickle the seed sequence
+    ss_ctor, ss_ctor_args, ss_state = seed_seq_info
+    seed_seq = ss_ctor(*ss_ctor_args)
+    seed_seq.__setstate__(ss_state)
```

Comment:
Should remove this here, for simplification.  Which you can do if you just use `self._seed_seq` in the `__reduce__` below (rather than `seed_seq.__reduce__()`).